compressor/tests/test_offline.py

```diff
@@ -30,7 +30,7 @@
 # The Jinja2 tests fail on Python 3.2 due to the following:
 # The line in compressor/management/commands/compress.py:
 #     compressor_nodes.setdefault(template, []).extend(nodes)
-# causes the error "unhashable type: 'Template'"
+# causes the error 'unhashable type: 'Template''
```

Comment:
Yeah, I also changed all the quotes to single quotes, because that's my own preference :-)